stories/initGraph.ts

```diff
@@ -0,0 +1,57 @@
+import { Graph } from '@stoplight/graph/dist/index.d';
+import { createFilesystemPlugin } from '@stoplight/graph/dist/plugins/filesystem';
+import { FilesystemTypes, IDirectory, IDirectoryInput } from '@stoplight/graph/dist/plugins/filesystem/types';
+import { createJsonPlugin } from '@stoplight/graph/dist/plugins/json';
+import { createOas2Plugin } from '@stoplight/graph/dist/plugins/oas/oas2';
+
+const petstore = require('./examples/oas2/petstore.json');
+
+import * as fs from 'fs';
+// @ts-ignore
+Buffer._useTypedArrays = true; // needed to avoid Uncaught TypeError: uint8BackedBuffer.toArrayBuffer is not a function
+
+const initFS = async () =>
```

Comment:
Nitpick: If you're not awaiting, you do not need to put the `async` keyword

Reply:
True, but then instead of just being like "yeah, it's async so I do `await initFS()`" you have to stop and look inside the function and think "huh, OK, this returns a Thenable so I need to do `await initFS()`". I like having it explicitly marked as async.